DEV-GUIDE.md

```diff
@@ -28,6 +28,8 @@ Also check [Troubleshooting](DEV-TROUBLESHOOTING.md).
   - [Adding dependencies](#adding-dependencies)
   - [About testing](#about-testing)
 - [Working with CircleCI](#working-with-circleci)
+- [Writing reference docs](#wiriting-reference-docs)
```

Comment:
writing

Reply:
This should be fixed. The link is broken.